lib/helpers.nix

```diff
@@ -97,5 +89,32 @@ let
     toLuaObject = helpers.lua.toLua;
     mkLuaInline = helpers.lua.mkInline;
   };
+
+  # Deprecated aliased for functions that rely on `pkgs`
```

Comment:
I think this needs a comment to say that all the attrs sets inside this nested attr set _need_ to be added in the manual "recursiveUpdate", in case we forget in 12 monts

Reply:
Hopefully it'll be gone in 12 months... But I agree let's add a comment.

Reply:
I added `` # WARNING: recursive merging must be handled manually; below in `self` ``